cardano-cli/src/Cardano/CLI/Types/Common.hs

```diff
@@ -227,9 +227,9 @@ instance FromJSON GenesisFile where
                            <> "Encountered: " <> show invalid
 
 -- | The desired output format.
-data PoolIdOutputFormat
-  = PoolIdOutputFormatHex
-  | PoolIdOutputFormatBech32
+data IdOutputFormat
+  = IdOutputFormatHex
+  | IdOutputFormatBech32
```

Comment:
This was renamed to remove `Pool` prefix because it can be used by `drep`s as well (and possibly others).

Reply:
Add a comment explicitly stating what this type is i.e an identifier for SPOs or DReps.

Reply:
Added a comment.